### PR TITLE
better tag line

### PR DESCRIPTION
**Pull request by marcoow**
bildschirmfoto 2017-04-11 um 14 28 18

### CODE REVIEW

**Comment by marcoow** [comment]
Hm, not actually sure _"move fast without breaking things"_ is something for the start page…

**Comment by gr0uch** [comment]
I think "Solid Software" clashes with something else: https://github.com/solid/

This is Tim Berners-Lee's project, not some unknown thing...

**Comment by marcoow** [comment]
Hm, not sure many people would not understand we mean the adjective tbh…

**Comment by gr0uch** [comment]
"Solid software for the web" out of context probably means TBL's project, that's what immediately came to mind.

**Comment by marcoow** [comment]
Hm, I'm not sure how it is out of context on our website?

**Comment by gr0uch** [comment]
Sorry if I'm not being clear: it implies that we are developing de-centralized web applications following the Solid conventions. Solid is a tech stack that has its own set of standards including RDF, WebID, FOAF, LDP, etc. To say "Solid" and "web software" in the same sentence is almost invariably going to mean TBL's project.

Somewhat related anecdote: former company of mine released an open-source project initially called "Unicorn", which clashed with the Rails web server `unicorn`. It seemed like a good idea internally to name it that even though it was well-known, but when people reacted to the name it didn't seem like a good idea after all. They changed the name of their project.

**Comment by marcoow** [comment]
I understand what you mean, I just don't see how that would be an association that would be more likely for anyone to make than the adjective.

**Comment by gr0uch** [comment]
>I just don't see how that would be an association that would be more likely for anyone to make than the adjective.

>To say "Solid" and "web software" in the same sentence is almost invariably going to mean TBL's project.

I feel strongly that this would be misleading and a synonym should be used, like "robust" (also Ember.js marketing word).

**Comment by marcoow** [comment]
btw: https://hashrocket.com/

**Comment by gr0uch** [comment]
They mention the technology used (Rails, React, etc) and follow up with "rock-solid", without mentioning the web at all.

i.e. "reactive conference" implies React.js, even though it has absolutely nothing to do with reactive programming techniques. "graph query language" implies GraphQL even though itself has nothing to do with graph data structures. Marketing logic!

**Comment by marcoow** [comment]
anyone else confused by _"solid"_ the adjective vs. "solid" the TBL project? I find this hard to follow tbh…

**Comment by oliverbarnes** [comment]
@daliwali thanks for the reference, I had never heard of the project, looks very interesting :)

I honestly doubt there'll be any confusion, Solid might be a great project, but not so ubiquitous to have any obvious association. My first association is just with the original meaning, and my second is with [SOLID the acronym for OO principles](https://en.wikipedia.org/wiki/SOLID_(object-oriented_design)), which isn't a bad one to have. But presumably it doesn't mean much in Elixir's functional world.

But I prefer Ambitious, I actually like the obvious association with Ember. We're a major Ember shop afterall, that's what we're known and what we get hired for. At least two major clients of ours are about truly ambitious products...

I have no idea how other Ember shops would react (no pun intended), though.


I'd run the options by client, ember people, etc, to get feedback

**Review by oliverbarnes — COMMENTED** [review]
@daliwali thanks for the reference, I had never heard of the project, looks very interesting :)

I honestly doubt there'll be any confusion, Solid might be a great project, but not so ubiquitous to have any obvious association. My first association is just with the original meaning, and my second is with [SOLID the acronym for OO principles](https://en.wikipedia.org/wiki/SOLID_(object-oriented_design)), which isn't a bad one to have. But presumably it doesn't mean much in Elixir's functional world.

But I prefer Ambitious, I actually like the obvious association with Ember. We're a major Ember shop afterall, that's what we're known and what we get hired for. At least two major clients of ours are about truly ambitious products...

I have no idea how other Ember shops would react (no pun intended), though.


I'd run the options by clients, ember people, etc, to get feedback

**Comment by gr0uch** [comment]
+1 to @oliverbarnes suggestion, "ambitious" sounds better, is closely associated with Ember.js, and is less ambiguous.

**Comment by marcoow** [comment]
> We're a major Ember shop afterall, that's what we're known and what we get hired for.

Good point but I think this text is mainly for people that don't know us anyway.

I updated the screenshot in the PR description to match the current status of this btw.

**Comment by oliverbarnes** [comment]
"For your business success" is a little stuffy, imo. Isn't it implicit?

How about "Solid software for your ambitious business"?

Or "Solid engineering for ambitious software"...

**Comment by marcoow** [comment]
Something like _"Solid engineering for ambitious software"_ sounds good to me. Maybe not exactly like that but I think that's a good direction - making clear we provide solid engineering and it makes most sense to hire us for ambitious projects… And then the smaller next line makes a bit more clear what it is we do.

**Comment by marcoow** [comment]
_"Solid engineering for ambitious projects"_ ?

**Review by gr0uch — APPROVED** [review]

**Review by pangratz — APPROVED** [review]

**Comment by oliverbarnes** [comment]
How about "...for clients around the world, from startups to long-established businesses"?

"conventional tools" may not convey the idea we want, of tools that follow conventions, but rather may sound like Microsoft Office-like tools. Not sure how to better word it... 

I'd change "we require our clients to take an active role" to something like "we ask our clients", or "we actively involve our clients in the development process"

**Comment by marcoow** [comment]
@oliverbarnes: _"conventions-based tools"_?

**Comment by oliverbarnes** [comment]
that works for me 👍
